app/shared/video-player/src/commonMain/kotlin/ui/state/PlayerState.kt

```diff
@@ -155,8 +156,17 @@ interface PlayerState {
     val audioTracks: TrackGroup<AudioTrack>
 
     fun saveScreenshotFile(filename: String)
+
+    val chapters: StateFlow<MutableList<Chapter>>
```

Comment:
```suggestion
    val chapters: StateFlow<List<Chapter>>
```
如无必要, 不要在接口使用 mutable 类型